### PR TITLE
NSXP-821: Updating README.md to include new nhf_summary_hexes_l1_actu…

### PR DESCRIPTION
**Pull request by Lexie-Ferry-NatureServe**
NSXP-821: Updating README.md to include new nhf_summary_hexes_l1_actual  layer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ nested_hexagon_framework_1sqm.gdb.zip contains a file geodatabase with the follo
 - nhf_summary_hexes_l2 - approximate hexagons for the Level 2 Summary Hex Aggregations
 - nhf_summary_hexes_l3 - approximate hexagons for the Level 3 Summary Hex Aggregations
 - grid_5deg_nhf_coverage - this shows the 5 degree grid zones covered by the original framework, those covered by NatureServe's extensions, and those which are not covered by either.
+- nhf_summary_hexes_l1_actual - The unioned geometries of the nhf_wheels layer grouped by the nhf_summary_hexes_l1_id. This is considered the "actual shape" of the nhf_summary_hexes_l1 geometries.
```
Comment (dhauverNS) [inline README.md]: How about: The actual shape of the Level 1 Summary Hex Aggregations, i.e., the union of the 343 individual hexagons found within the wheel features which compose the Level 1 Summary Hex Aggregation.

Comment (Lexie-Ferry-NatureServe) [reply README.md]: Works for me. I'll update here and in the README on the S: drive.

Comment (Lexie-Ferry-NatureServe) [reply README.md]: @dhauverNS I've pushed up a corrected version.

**Review by dhauverNS — APPROVED** [review]
Looks good, but we should hold off on merging until we're ready to create the release and upload its artifacts.

**Pull request merged**